stdlib/2/SocketServer.pyi

```diff
@@ -1,22 +1,22 @@
 # NB: SocketServer.pyi and socketserver.pyi must remain consistent!
 # Stubs for socketserver
 
-from typing import Any, BinaryIO, Optional, Tuple, Type, Text, Union
+from typing import Any, BinaryIO, Callable, Optional, Tuple, Type, Text, Union
 from socket import SocketType
 import sys
 import types
 
 class BaseServer:
     address_family: int
-    RequestHandlerClass: type
+    RequestHandlerClass: Callable[..., BaseRequestHandler]
```

Comment:
This risks triggering python/mypy#708 unfortunately. How is this attribute typically used?

Reply:
I think code is unlikely to be assigning to this attribute directly as it's initalized in `__init__` with a required argument. I can change this line back, though, if you want. The primary problem I wanted to fix was the initializer needing to take a callable rather than the type itself.